public/docs/_examples/template-syntax/dart/lib/app_component.html

```diff
@@ -583,7 +583,7 @@ <h4 id="ngFor-trackBy">NgForTrackBy</h4>
 </div>
 
 <p>with trackBy and <i>space</i> separator</p>
-<div class="box">
+<div #withTrackBy class="box">
```

Comment:
Remove `#withTrackBy` here, it's used on the preceding `div` to compare the number of changes with the equivalent `ngFor` without using `trackBy`.

When adding `#withTrackBy` here, you'll be counting the changes twice.